assets/prefabs/combat/items/bulletItem.prefab

```diff
@@ -7,7 +7,7 @@
         "icon" : "MetalRenegades:bulletIcon",
         "usage": "IN_DIRECTION",
         "stackId": "MetalRenegades:bulletItem",
-        "cooldownTime": 0
+        "cooldownTime": 300
```

Comment:
Increasing the cooldown is definitely required. Maybe look into changing the damage too?